invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookMetadata/BookMetadata.js

```diff
@@ -0,0 +1,275 @@
+import React, { Component } from 'react';
+import {
+  Grid,
+  Segment,
+  Image,
+  Responsive,
+  Container,
+  Button,
+  Header,
+  List,
+  Icon,
+} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import BookTab from '../BookTab';
+import { RequestNewLoanForm } from './components/RequestNewLoanForm';
+import '../../BookDetails.scss';
+
+export default class BookMetadata extends Component {
+  state = { visible: false };
+
+  toggleVisibility = () => this.setState({ visible: !this.state.visible });
+
+  _render_book_info() {
+    const bookData = this.props.bookDetails;
+    return (
+      <div className="book-info">
+        <Header as="h2">{bookData.title}</Header>
+        <List>
+          {bookData.authors.map((author, index) => (
+            <List.Item as="h4" key={`Key${index}`}>
+              Author: {author}
+            </List.Item>
+          ))}
+        </List>
+        <List>
+          {bookData.publishers.map((publisher, index) => (
+            <List.Item as="h5" key={`Key${index}`}>
+              Publisher: {publisher}
```

Comment:
did we have the Publisher: label in the mockup? Could you please add screenshots of the detail page to the PR?